test/multiapi/Expected/AcceptanceTests/MultiapiCustomBaseUrl/multiapicustombaseurl/_multiapi_custom_base_url_service_client.py

```diff
@@ -11,9 +11,8 @@
 
 from typing import TYPE_CHECKING
 
+from ._profiles import KnownProfiles, MultiApiClientMixin, ProfileDefinition
```

Comment:
You can see on line 52 that I'm not changing the docstrings of profiles. I thought that would expose my vendored file too much